apps/docs/astro.config.ts

```diff
@@ -4,6 +4,7 @@ import { resolve } from 'path'
 import preprocess from 'svelte-preprocess'
 import rehypeSlug from 'rehype-slug'
 import rehypeAutolinkHeadings from 'rehype-autolink-headings'
+import mkcert from 'vite-plugin-mkcert'
```

Comment:
Ended up adding this plugin to be able to debug docs examples locally. Let me know if this is a problem!